spec.md

```diff
@@ -613,6 +613,12 @@ This RPC will be called by the CO to provision a new volume on behalf of a user
 This operation MUST be idempotent.
 If a volume corresponding to the specified volume `name` already exists, is accessible from `accessibility_requirements`, and is compatible with the specified `capacity_range`, `volume_capabilities` and `parameters` in the `CreateVolumeRequest`, the Plugin MUST reply `0 OK` with the corresponding `CreateVolumeResponse`.
 
+Plugins MAY create 3 types of volumes:
+
+- Empty volumes. When plugin supports `CREATE_DELETE_VOLUME` optional capability.
+- From an existing snapshot. When plugin supports `CREATE_DELETE_SNAPSHOT` optional capability.
+- From an existing volume. When plugin supports efficient cloning, and reports the optional capability `CLONE_VOLUME`.
```

Comment:
nit: why is "efficient" a prerequisite for this capability? either a plugin has the cap or it doesn't.

Reply:
A plugin could do host assisted "cloning" by creating a new volume, attaching both volumes, and `dd`-ing contents from source to destination.

I don't think it would be a good idea mixing both concepts (fast storage backend cloning and slow host assisted cloning) under the same capability.  That would probably lead to unfulfilled expectations from the caller side.

We could add two capabilities instead of one, `CLONE_VOLUME` and `EFFICIENT_CLONING`. 
 That way plugins for backends supporting efficient cloning would report them both, and those doing host assisted cloning would just report the `CLONE_VOLUME` capability.  But my initial though is that this would be more confusing/troublesome.

Reply:
So I think that in discussions we agreed that we didn't need/want to specify here and wanted to keep this generic in terms of the implementation with the caveat that if we need to we could consider additions for host assisted type cloning in the future?  If folks agree it'd be great to remove the specifics here and see if we can move forward with getting this merged.

Reply:
Thanks, I removed "efficient" from here, and added a comment below on the attaching and copying as agreed in the meeting.